src/website/src/pages/ec/utilities/spacing/docs/usage.mdx

```diff
@@ -16,13 +16,13 @@ Different clases can be used to set the margin around an element. All spacing to
 
 Available values are:
 
-- `ecl-u-ma-*` (ecl-u-ma-2xs to ecl-u-ma-4xl): all around margins
-- `ecl-u-mv-*` (ecl-u-mv-2xs to ecl-u-mv-4xl): vertical margins
-- `ecl-u-mh-*` (ecl-u-mh-2xs to ecl-u-mh-4xl): horizontal margins
-- `ecl-u-mt-*` (ecl-u-mt-2xs to ecl-u-mt-4xl): top margins
-- `ecl-u-mr-*` (ecl-u-mr-2xs to ecl-u-mr-4xl): right margins
-- `ecl-u-mb-*` (ecl-u-mb-2xs to ecl-u-mb-4xl): bottom margins
-- `ecl-u-ml-*` (ecl-u-ml-2xs to ecl-u-ml-4xl): left margins
+- `ecl-u-ma-*` (ecl-u-ma-2xs to ecl-u-ma-6xl): all around margins
```

Comment:
reading this docs i went then checking the utilities we generate, thinking that maybe we were still using left or right there, but instead we do use inline-start and inline-end, so why don't we advertise this anywhere? I think we can still use the same "labels" like "left margins" for clarity, but on top we can specify that our spacing utilities are RTL compliant.